src/lib/Behat/Component/ContentTypePicker.php

```diff
@@ -44,6 +44,7 @@ protected function specifyLocators(): array
             new VisibleCSSLocator('filterInput', '.ibexa-extra-actions__section-content--content-type .ibexa-instant-filter__input'),
             new VisibleCSSLocator('filteredItem', '.ibexa-extra-actions__section-content--content-type .ibexa-instant-filter__group-item:not([hidden]) .form-check-label'),
             new VisibleCSSLocator('header', '.ibexa-extra-actions--create .ibexa-extra-actions__header h2'),
+            new VisibleCSSLocator('languageSelectWhileCreatingItem', '#content_create_language'),
```

Comment:
Should be removed - but dependency on IbexaDropdown and the `selectLanguage` should be added here